src/duqtools/cli.py

```diff
@@ -384,10 +384,54 @@ def cli_dash(**kwargs):
 
 
 @cli.command('merge')
-@click.option('--all', 'merge_all', is_flag=True, help='Try to merge all known variables.')
+@click.option('-t',
+              '--template',
+              required=True,
+              type=str,
+              help='IMAS location to use as the template for the target')
+@click.option('-T',
+              '--target',
+              required=True,
+              type=str,
+              help='IMAS location to store the result in')
+@click.option('-h',
+              '--handle',
+              'handles',
+              type=str,
+              help='handles to merge to the target',
+              multiple=True)
```

Comment:
In `duqtools plot` we use this as the interface:
```
  -m, --imas TEXT     IMAS path formatted as <user>/<db>/<shot>/<number>.
  -u, --user TEXT     IMAS user.
  -d, --db TEXT       IMAS database.
  -s, --shot INTEGER  IMAS shot.
  -r, --runs INTEGER  IMAS run (multiple runs can be specified)
  -i, --input TEXT    Input file, i.e. `data.csv` or `runs.yaml`
```

I think we should aim for consistency here. Maybe this tools should not have the option to define separate handles and just take a `data.csv` as input.

Reply:
I think `plot` should be updated then, it does not make sense to have seperate options for a single IMAS path

Reply:
the effect of the '-r' flag can also be achieved with:
> -h jet/1234/80{01,02,03,04,05,06,07}

Reply:
Sounds good, I will make an issue for this.